docs/mds/ARCHITECTURE_DOCUMENT.md

```diff
@@ -183,237 +184,305 @@ Para o projeto será utilizado o PostgreSQL como o banco de dados de desenvolvim
+4 - Dentro do **Django** a requisição recebida pelo **web server** é mapeada para uma view específica através das urls. Essa view pode ser tanto de aplicações do projeto Dr. Down como aplicações externas. Elas pedem dados as modelos, as modelos do Dr. Down fazem uma requisição ao **redis** que pega os dados do banco de dados **Postgresql** e retorna a view, a view seleciona o template e fornece os dados. Assim, o template é preenchido e devolvido a view, que devolve o template como resposta ao web server.
 
-5 - O web server (nginx) retorna a resposta para o web client (navegador)
+5 - O web server (Nginx) retorna a resposta para o web client (navegador)
 
 ## 3:  Requisitos e Restrições Arquiteturais
 
 ### 3.1 Dr. Down
 
-Linguagem: Python 3.6.4
+| Requisito | Ferramenta/Solução |
+|---|---|
+|Linguagem| Python 3.6.4 |
+|Framework| Django 2.0.3 |
+|Plataforma| Web - Navegadores Google Chrome, Safari e Firefox |
+|Segurança | O sistema terá informações pessoais dos pacientes que só poderão ser vistas pelo mesmo ou pelo(s) seu(s) respectivo(s) médico(s). Outros dados pessoais só poderão ser vistos pelo próprio usuário.
+|Internacionalização (i18n)|	A aplicação terá suporte aos idiomas: Inglês e Português do Brasil (sendo esta a linguagem padrão).
 
-Framework: Django 2.0.3
+### 3.2 Docker e Compose
 
-Plataforma: Web - Navegadores Google Chrome, Safari e Firefox
+| Nome | Versão |
+|---|---|
+| Docker | 1.13.1
+| Docker Compose | 1.8.0
 
-Segurança: O sistema terá informações pessoais dos pacientes que só poderão ser vistas pelo mesmo ou pelo(s) seu(s) respectivo(s) médico(s). Outros dados pessoais só poderão ser vistos pelo próprio usuário.
+## 4:	Visão Lógica
 
-Internacionalização (i18n):	A aplicação terá suporte aos idiomas: Inglês e Português do Brasil (sendo esta a linguagem padrão).
+### 4.1	Visão Geral: Pacotes e Camadas
 
-### 3.2 Docker e Compose
+O framework Django organiza os projetos em apps, que são pastas que contêm uma funcionalidade independente do restante da aplicação. Além disso, existem arquivos de configuração e arquivos estáticos globais. A figura a seguir mostra a organização de pastas de um app.
 
-Docker versão: 1.13.1
+![Diagrama de Pacotes](http://uploaddeimagens.com.br/images/001/379/389/original/pacotes.jpeg?1524078223)
 
-Docker Compose versão: 1.8.0
+- **apps**: cada app tem uma pasta com as suas models, views, formulários e testes. Além disso, também há um arquivo URLs que será incluso no URLs global.
 
-## 4:	Visão Lógica
+- **static** : arquivos estáticos aplicados em todas os apps.
```

Comment:
Não existe uma pasta static aplicado a todos os apps. é uma pasta static para cada app ai se rodar o comando collectstatic ele gera uma pasta chamado staticfiles com todos os arquivos staticos de todos os apps, mas isso é só pra produção, não entra no diagrama

Reply:
Os estaticos da homepage ficam na externa. Se não existir, não tem home. Aí faz o que?

Reply:
teria que criar um app chamado core só para coisas assim

Reply:
mas não tem um app chamado homepage?

Reply:
Tem não. Se for fazer a core vai ter que refazer o projeto basicamente. Remapear tudo

Reply:
Homepage tá sem app